### ISSUE TITLE
false positive for unbalanced-tuple-unpacking

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#329]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

e.g. when implementing variadic functions

```
#!python
def func(*args):
    if len(args) == 1:
        arg1, = args
```

This raises `Possible unbalanced tuple unpacking with sequence: left side has 1 label(s), right side has 0 value(s) (unbalanced-tuple-unpacking)`

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/329
